### ISSUE TITLE
[4.x] MigrationException on fresh install

### ISSUE THREAD

**Issue by jimirobaer** [verbb/formie#846]
**Description**

I am unable to install Formie on a fresh install of Craft CMS 4.0.0-beta4. I get a MigrationExpection caused by something related to the stencils.

**Steps to reproduce**
1. Install Craft CMS 4.0.0-beta4
2. Install Formie through Plugin Store

**Additional info**
- Plugin version: 2.0.0-beta.2
- Craft version: 4.0.0-beta4
- Multi-site: No

**Additional context**

We are preparing an update to our starter template to Craft 4.0. I'm not sure it makes sense to create (many) bug reports at this time for releases (Craft CMS + plugins) that are not yet stable. Be sure to let me know.

Stack Trace

```
Error: Typed property verbb\formie\models\Stencil::$submitActionEntryId must not be accessed before initialization in craft4-vanilla/vendor/verbb/formie/src/models/Stencil.php:284
Stack trace:
#0 craft4-vanilla/vendor/verbb/formie/src/models/Stencil.php(239): verbb\formie\models\Stencil->getRedirectEntry()
#1 craft4-vanilla/vendor/verbb/formie/src/services/Stencils.php(172): verbb\formie\models\Stencil->getConfig()
#2 craft4-vanilla/vendor/verbb/formie/src/migrations/Install.php(555): verbb\formie\services\Stencils->saveStencil(Object(verbb\formie\models\Stencil))
#3 craft4-vanilla/vendor/verbb/formie/src/migrations/Install.php(464): verbb\formie\migrations\Install->_defaultStencils()
#4 craft4-vanilla/vendor/verbb/formie/src/migrations/Install.php(34): verbb\formie\migrations\Install->insertDefaultData()
#5 craft4-vanilla/vendor/craftcms/cms/src/db/Migration.php(49): verbb\formie\migrations\Install->safeUp()
#6 craft4-vanilla/vendor/craftcms/cms/src/db/MigrationManager.php(233): craft\db\Migration->up(true)
#7 craft4-vanilla/vendor/craftcms/cms/src/base/Plugin.php(136): craft\db\MigrationManager->migrateUp(Object(verbb\formie\migrations\Install))
#8 craft4-vanilla/vendor/craftcms/cms/src/services/Plugins.php(533): craft\base\Plugin->install()
#9 craft4-vanilla/vendor/craftcms/cms/src/controllers/PluginsController.php(48): craft\services\Plugins->installPlugin('formie', 'standard')
#10 [internal function]: craft\controllers\PluginsController->actionInstallPlugin()
#11 craft4-vanilla/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#12 craft4-vanilla/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#13 craft4-vanilla/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('install-plugin', Array)
#14 craft4-vanilla/vendor/craftcms/cms/src/web/Application.php(301): yii\base\Module->runAction('plugins/install...', Array)
#15 craft4-vanilla/vendor/craftcms/cms/src/web/Application.php(625): craft\web\Application->runAction('plugins/install...', Array)
#16 craft4-vanilla/vendor/craftcms/cms/src/web/Application.php(280): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#17 craft4-vanilla/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#18 craft4-vanilla/web/index.php(12): yii\base\Application->run()
#19 .composer/vendor/laravel/valet/server.php(234): require('/Users/...')
#20 {main}

Next craft\errors\MigrationException: An error occurred while executing the "verbb\formie\migrations\Install migration: Typed property verbb\formie\models\Stencil::$submitActionEntryId must not be accessed before initialization in craft4-vanilla/vendor/craftcms/cms/src/db/MigrationManager.php:253
Stack trace:
#0 craft4-vanilla/vendor/craftcms/cms/src/base/Plugin.php(136): craft\db\MigrationManager->migrateUp(Object(verbb\formie\migrations\Install))
#1 craft4-vanilla/vendor/craftcms/cms/src/services/Plugins.php(533): craft\base\Plugin->install()
#2 craft4-vanilla/vendor/craftcms/cms/src/controllers/PluginsController.php(48): craft\services\Plugins->installPlugin('formie', 'standard')
#3 [internal function]: craft\controllers\PluginsController->actionInstallPlugin()
#4 craft4-vanilla/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#5 craft4-vanilla/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#6 craft4-vanilla/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('install-plugin', Array)
#7 craft4-vanilla/vendor/craftcms/cms/src/web/Application.php(301): yii\base\Module->runAction('plugins/install...', Array)
#8 craft4-vanilla/vendor/craftcms/cms/src/web/Application.php(625): craft\web\Application->runAction('plugins/install...', Array)
#9 craft4-vanilla/vendor/craftcms/cms/src/web/Application.php(280): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#10 craft4-vanilla/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#11 craft4-vanilla/web/index.php(12): yii\base\Application->run()
#12 .composer/vendor/laravel/valet/server.php(234): require('/Users/...')
#13 {main}

```

**Comment by engram-design** [comment]
Thanks for the heads-up, and yep do feel free to send any bugs with the beta out, as we're getting pretty close to finalising things. This I believe was just a incorrect update from the `craft-3` branch. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.2",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.3](https://github.com/verbb/formie/releases/tag/2.0.0-beta.3)
